src/indicators/drought/iri_rainfallforecast.py

```diff
@@ -43,44 +41,79 @@ def download_iri(iri_auth, config, chunk_size=128):
     # couldn't figure how to improve it
     logger.info("Downloading IRI NetCDF file. This might take some time")
     response = requests.get(config.IRI_URL, cookies=cookies, verify=False)
-    with open(IRI_filepath, "wb") as fd:
+    with open(output_path, "wb") as fd:
         for chunk in response.iter_content(chunk_size=chunk_size):
             fd.write(chunk)
 
-    # TODO: explore if can also open with rioxarray instead of xarray.
-    # Getting an error with current settings
+
+def clean_iri_data(
```

Comment:
A general comment, but have you been in touch with IRI about these data problems? Maybe they want to take on some of this processing internally to improve their data quality?

Reply:
Hmm no, nice idea, I will